### PR TITLE
Publish ARM Ansible Docker images (`future` tag)

### PR DESCRIPTION
**Pull request by peterdeme**
## Description of the change

Publish ARM Ansible Docker images, plus additional sanity test.

- This PR must be merged first: https://github.com/spacelift-io/runner-terraform/pull/28

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue);
- [x] New feature (non-breaking change that adds functionality);
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected);
- [ ] Other (miscellaneous, GitHub workflow changes, changes to the PR template);

### Development

- [ ] Parts of this pull request impacting core (user-facing, documented) product functionality have test coverage;

### Code review

- [ ] This pull request has a descriptive title and sufficient context for a reviewer. There may be a screenshot or screencast attached;
- [ ] This pull request is no longer a draft;
- [ ] You have reviewed this Pull Request yourself;

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -46,23 +52,32 @@ jobs:
...
         with:
           context: base
+          platforms: linux/amd64,linux/arm64
           push: ${{ github.ref == 'refs/heads/main' || github.ref == 'refs/heads/future' }}
           tags: |
             ${{ secrets.PUBLIC_RUNNER_ANSIBLE_ECR_REPOSITORY_URL }}:${{ github.ref == 'refs/heads/main' && 'latest' || 'future' }}
+            ghcr.io/spacelift-io/runner-ansible:${{ github.ref == 'refs/heads/main' && 'latest' || 'future' }}
 
   aws:
+    if: false # skip aws build until we push base image to the repo
```
Comment (peterdeme) [inline .github/workflows/deploy.yml:68]: is this line still needed @michalg9? I don't think so

Comment (michalg9) [reply .github/workflows/deploy.yml:68]: Nope, that was just for initial bootstrapping (when we did not have base image in AWS)

Comment (peterdeme) [reply .github/workflows/deploy.yml:68]: I thought so. Removing it.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -2,47 +2,52 @@ name: Build
...
             REPOSITORY_BASE_PATH=${{ secrets.PUBLIC_RUNNER_ANSIBLE_ECR_REPOSITORY_URL }}
-          context: aws
           push: false
-          tags: |
-            ${{ secrets.PREPROD_PUBLIC_RUNNER_ANSIBLE_ECR_REPOSITORY_URL_AWS }}:${{ github.ref == 'refs/heads/main' && 'latest' || 'future' }}
+          load: true
+          platforms: linux/${{ matrix.arch }}
+          tags: ${{ env.IMAGE_NAME }}
+
+      - name: Test if ansible and Python binary works
```
Comment (michalg9) [inline .github/workflows/build.yml:47]: Could we test `ansible-playbook`, `ansible-galaxy` and `ansible-runner` binaries here as well?

**Review by michalg9 — APPROVED** [review]
Looks good, just left one nit related to testing binaries

**Pull request merged**